ggml/src/ggml-kompute.cpp

```diff
@@ -2006,7 +2033,7 @@ ggml_backend_t ggml_backend_kompute_init(int device) {
     ggml_backend_t kompute_backend = new ggml_backend {
         /* .guid      = */ ggml_backend_kompute_guid(),
         /* .interface = */ kompute_backend_i,
-        /* .device    = */ nullptr,
+        /* .device    = */ ggml_backend_reg_dev_get(ggml_backend_kompute_reg(), 0),
```

Comment:
It's important that the device is set to the correct one

Reply:
Fixed, thanks.